### PR TITLE
Fix 11327

### LINKED ISSUE #11327: Cannot convert column `now64()` because it is constant but values of constants are different in source and result.
```
clickhouse benchmark <<< "SELECT now64() FROM remote('127.0.0.1', 'system', 'numbers_mt') WHERE number<100000 LIMIT 100000"
Loaded 1 queries.
An error occurred while processing query:
SELECT now64() FROM remote('127.0.0.1', 'system', 'numbers_mt') WHERE number<100000 LIMIT 100000
DB::Exception: Received from localhost:9000. DB::Exception: Cannot convert column `now64()` because it is constant but values of constants are different in source and result. Stack trace:

0. /build/obj-x86_64-linux-gnu/../contrib/poco/Foundation/src/Exception.cpp:27: Poco::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int) @ 0x1065a1f0 in /usr/lib/debug/usr/bin/clickhouse
1. /build/obj-x86_64-linux-gnu/../src/Common/Exception.cpp:32: DB::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int) @ 0x914dd3d in /usr/lib/debug/usr/bin/clickhouse
2. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/string:2134: DB::ConvertingTransform::ConvertingTransform(DB::Block, DB::Block, DB::ConvertingTransform::MatchColumnsMode) (.cold) @ 0xe0f1a6b in /usr/lib/debug/usr/bin/clickhouse
3. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:2214: std::__1::__function::__func<DB::ClusterProxy::(anonymous namespace)::createLocalStream(std::__1::shared_ptr<DB::IAST> const&, DB::Block const&, DB::Context const&, DB::QueryProcessingStage::Enum, bool, bool, bool)::'lambda'(DB::Block const&), std::__1::allocator<DB::ClusterProxy::(anonymous namespace)::createLocalStream(std::__1::shared_ptr<DB::IAST> const&, DB::Block const&, DB::Context const&, DB::QueryProcessingStage::Enum, bool, bool, bool)::'lambda'(DB::Block const&)>, std::__1::shared_ptr<DB::IProcessor> (DB::Block const&)>::operator()(DB::Block const&) @ 0xda854f2 in /usr/lib/debug/usr/bin/clickhouse
4. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:3821: void DB::QueryPipeline::addSimpleTransformImpl<std::__1::function<std::__1::shared_ptr<DB::IProcessor> (DB::Block const&)> >(std::__1::function<std::__1::shared_ptr<DB::IProcessor> (DB::Block const&)> const&)::'lambda'(DB::OutputPort*&, DB::QueryPipeline::StreamType, unsigned long)::operator()(DB::OutputPort*&, DB::QueryPipeline::StreamType, unsigned long) const @ 0xdf1ea77 in /usr/lib/debug/usr/bin/clickhouse
5. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/vector:656: void DB::QueryPipeline::addSimpleTransformImpl<std::__1::function<std::__1::shared_ptr<DB::IProcessor> (DB::Block const&)> >(std::__1::function<std::__1::shared_ptr<DB::IProcessor> (DB::Block const&)> const&) @ 0xdf1f221 in /usr/lib/debug/usr/bin/clickhouse
6. /build/obj-x86_64-linux-gnu/../src/Interpreters/ClusterProxy/SelectStreamFactory.cpp:103: DB::ClusterProxy::(anonymous namespace)::createLocalStream(std::__1::shared_ptr<DB::IAST> const&, DB::Block const&, DB::Context const&, DB::QueryProcessingStage::Enum, bool, bool, bool) @ 0xda86529 in /usr/lib/debug/usr/bin/clickhouse
7. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/vector:1681: DB::ClusterProxy::SelectStreamFactory::createForShard(DB::Cluster::ShardInfo const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::shared_ptr<DB::IAST> const&, DB::Context const&, std::__1::shared_ptr<DB::Throttler> const&, DB::SelectQueryInfo const&, std::__1::vector<DB::Pipe, std::__1::allocator<DB::Pipe> >&)::'lambda'()::operator()() const @ 0xda86707 in /usr/lib/debug/usr/bin/clickhouse
8. /build/obj-x86_64-linux-gnu/../src/Interpreters/ClusterProxy/SelectStreamFactory.cpp:212: DB::ClusterProxy::SelectStreamFactory::createForShard(DB::Cluster::ShardInfo const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::shared_ptr<DB::IAST> const&, DB::Context const&, std::__1::shared_ptr<DB::Throttler> const&, DB::SelectQueryInfo const&, std::__1::vector<DB::Pipe, std::__1::allocator<DB::Pipe> >&) @ 0xda878cd in /usr/lib/debug/usr/bin/clickhouse
9. /build/obj-x86_64-linux-gnu/../src/Interpreters/ClusterProxy/executeQuery.cpp:71: DB::ClusterProxy::executeQuery(DB::ClusterProxy::IStreamFactory&, std::__1::shared_ptr<DB::Cluster> const&, std::__1::shared_ptr<DB::IAST> const&, DB::Context const&, DB::Settings const&, DB::SelectQueryInfo const&) @ 0xda8e678 in /usr/lib/debug/usr/bin/clickhouse
10. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/__tree:1085: DB::StorageDistributed::read(std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, DB::SelectQueryInfo const&, DB::Context const&, DB::QueryProcessingStage::Enum, unsigned long, unsigned int) @ 0xda78a50 in /usr/lib/debug/usr/bin/clickhouse
11. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/vector:1383: void DB::InterpreterSelectQuery::executeFetchColumns<DB::QueryPipeline>(DB::QueryProcessingStage::Enum, DB::QueryPipeline&, std::__1::shared_ptr<DB::PrewhereInfo> const&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, DB::QueryPipeline&) @ 0xd67ed2b in /usr/lib/debug/usr/bin/clickhouse
12. /build/obj-x86_64-linux-gnu/../src/Interpreters/InterpreterSelectQuery.cpp:788: void DB::InterpreterSelectQuery::executeImpl<DB::QueryPipeline>(DB::QueryPipeline&, std::__1::shared_ptr<DB::IBlockInputStream> const&, std::__1::optional<DB::Pipe>, DB::QueryPipeline&) @ 0xd6824c2 in /usr/lib/debug/usr/bin/clickhouse
13. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/optional:224: DB::InterpreterSelectQuery::executeWithProcessors() @ 0xd6464fd in /usr/lib/debug/usr/bin/clickhouse
14. /build/obj-x86_64-linux-gnu/../src/Interpreters/InterpreterSelectWithUnionQuery.cpp:256: DB::InterpreterSelectWithUnionQuery::executeWithProcessors() @ 0xd817c75 in /usr/lib/debug/usr/bin/clickhouse
15. /build/obj-x86_64-linux-gnu/../src/Interpreters/executeQuery.cpp:341: DB::executeQueryImpl(char const*, char const*, DB::Context&, bool, DB::QueryProcessingStage::Enum, bool, DB::ReadBuffer*, bool) @ 0xd956d94 in /usr/lib/debug/usr/bin/clickhouse
16. /build/obj-x86_64-linux-gnu/../src/Interpreters/executeQuery.cpp:588: DB::executeQuery(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, DB::Context&, bool, DB::QueryProcessingStage::Enum, bool, bool) @ 0xd95a735 in /usr/lib/debug/usr/bin/clickhouse
17. /build/obj-x86_64-linux-gnu/../programs/server/TCPHandler.cpp:252: DB::TCPHandler::runImpl() @ 0x9256a98 in /usr/lib/debug/usr/bin/clickhouse
18. /build/obj-x86_64-linux-gnu/../programs/server/TCPHandler.cpp:1244: DB::TCPHandler::run() @ 0x9257a70 in /usr/lib/debug/usr/bin/clickhouse
19. /build/obj-x86_64-linux-gnu/../contrib/poco/Net/src/TCPServerConnection.cpp:57: Poco::Net::TCPServerConnection::start() @ 0x1057801b in /usr/lib/debug/usr/bin/clickhouse
20. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/atomic:856: Poco::Net::TCPServerDispatcher::run() @ 0x105784ab in /usr/lib/debug/usr/bin/clickhouse
21. /build/obj-x86_64-linux-gnu/../contrib/poco/Foundation/include/Poco/Mutex_POSIX.h:59: Poco::PooledThread::run() @ 0x106f6fd6 in /usr/lib/debug/usr/bin/clickhouse
22. /build/obj-x86_64-linux-gnu/../contrib/poco/Foundation/include/Poco/AutoPtr.h:223: Poco::ThreadImpl::runnableEntry(void*) @ 0x106f23d0 in /usr/lib/debug/usr/bin/clickhouse
23. start_thread @ 0x76db in /lib/x86_64-linux-gnu/libpthread-2.27.so
24. /build/glibc-OTsEL5/glibc-2.27/misc/../sysdeps/unix/sysv/linux/x86_64/clone.S:97: __clone @ 0x12188f in /usr/lib/debug/lib/x86_64-linux-gnu/libc-2.27.so

Exception in ThreadPool(max_threads: 1, max_free_threads: 1, queue_size: 1, shutdown_on_exception: 1).: Code: 171, e.displayText() = DB::Exception: Received from localhost:9000. DB::Exception: Cannot convert column `now64()` because it is constant but values of constants are different in source and result. Stack trace:

0. /build/obj-x86_64-linux-gnu/../contrib/poco/Foundation/src/Exception.cpp:27: Poco::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int) @ 0x1065a1f0 in /usr/lib/debug/usr/bin/clickhouse
1. /build/obj-x86_64-linux-gnu/../src/Common/Exception.cpp:32: DB::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int) @ 0x914dd3d in /usr/lib/debug/usr/bin/clickhouse
2. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/string:2134: DB::ConvertingTransform::ConvertingTransform(DB::Block, DB::Block, DB::ConvertingTransform::MatchColumnsMode) (.cold) @ 0xe0f1a6b in /usr/lib/debug/usr/bin/clickhouse
3. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:2214: std::__1::__function::__func<DB::ClusterProxy::(anonymous namespace)::createLocalStream(std::__1::shared_ptr<DB::IAST> const&, DB::Block const&, DB::Context const&, DB::QueryProcessingStage::Enum, bool, bool, bool)::'lambda'(DB::Block const&), std::__1::allocator<DB::ClusterProxy::(anonymous namespace)::createLocalStream(std::__1::shared_ptr<DB::IAST> const&, DB::Block const&, DB::Context const&, DB::QueryProcessingStage::Enum, bool, bool, bool)::'lambda'(DB::Block const&)>, std::__1::shared_ptr<DB::IProcessor> (DB::Block const&)>::operator()(DB::Block const&) @ 0xda854f2 in /usr/lib/debug/usr/bin/clickhouse
4. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:3821: void DB::QueryPipeline::addSimpleTransformImpl<std::__1::function<std::__1::shared_ptr<DB::IProcessor> (DB::Block const&)> >(std::__1::function<std::__1::shared_ptr<DB::IProcessor> (DB::Block const&)> const&)::'lambda'(DB::OutputPort*&, DB::QueryPipeline::StreamType, unsigned long)::operator()(DB::OutputPort*&, DB::QueryPipeline::StreamType, unsigned long) const @ 0xdf1ea77 in /usr/lib/debug/usr/bin/clickhouse
5. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/vector:656: void DB::QueryPipeline::addSimpleTransformImpl<std::__1::function<std::__1::shared_ptr<DB::IProcessor> (DB::Block const&)> >(std::__1::function<std::__1::shared_ptr<DB::IProcessor> (DB::Block const&)> const&) @ 0xdf1f221 in /usr/lib/debug/usr/bin/clickhouse
6. /build/obj-x86_64-linux-gnu/../src/Interpreters/ClusterProxy/SelectStreamFactory.cpp:103: DB::ClusterProxy::(anonymous namespace)::createLocalStream(std::__1::shared_ptr<DB::IAST> const&, DB::Block const&, DB::Context const&, DB::QueryProcessingStage::Enum, bool, bool, bool) @ 0xda86529 in /usr/lib/debug/usr/bin/clickhouse
7. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/vector:1681: DB::ClusterProxy::SelectStreamFactory::createForShard(DB::Cluster::ShardInfo const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::shared_ptr<DB::IAST> const&, DB::Context const&, std::__1::shared_ptr<DB::Throttler> const&, DB::SelectQueryInfo const&, std::__1::vector<DB::Pipe, std::__1::allocator<DB::Pipe> >&)::'lambda'()::operator()() const @ 0xda86707 in /usr/lib/debug/usr/bin/clickhouse
8. /build/obj-x86_64-linux-gnu/../src/Interpreters/ClusterProxy/SelectStreamFactory.cpp:212: DB::ClusterProxy::SelectStreamFactory::createForShard(DB::Cluster::ShardInfo const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::shared_ptr<DB::IAST> const&, DB::Context const&, std::__1::shared_ptr<DB::Throttler> const&, DB::SelectQueryInfo const&, std::__1::vector<DB::Pipe, std::__1::allocator<DB::Pipe> >&) @ 0xda878cd in /usr/lib/debug/usr/bin/clickhouse
9. /build/obj-x86_64-linux-gnu/../src/Interpreters/ClusterProxy/executeQuery.cpp:71: DB::ClusterProxy::executeQuery(DB::ClusterProxy::IStreamFactory&, std::__1::shared_ptr<DB::Cluster> const&, std::__1::shared_ptr<DB::IAST> const&, DB::Context const&, DB::Settings const&, DB::SelectQueryInfo const&) @ 0xda8e678 in /usr/lib/debug/usr/bin/clickhouse
10. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/__tree:1085: DB::StorageDistributed::read(std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, DB::SelectQueryInfo const&, DB::Context const&, DB::QueryProcessingStage::Enum, unsigned long, unsigned int) @ 0xda78a50 in /usr/lib/debug/usr/bin/clickhouse
11. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/vector:1383: void DB::InterpreterSelectQuery::executeFetchColumns<DB::QueryPipeline>(DB::QueryProcessingStage::Enum, DB::QueryPipeline&, std::__1::shared_ptr<DB::PrewhereInfo> const&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, DB::QueryPipeline&) @ 0xd67ed2b in /usr/lib/debug/usr/bin/clickhouse
12. /build/obj-x86_64-linux-gnu/../src/Interpreters/InterpreterSelectQuery.cpp:788: void DB::InterpreterSelectQuery::executeImpl<DB::QueryPipeline>(DB::QueryPipeline&, std::__1::shared_ptr<DB::IBlockInputStream> const&, std::__1::optional<DB::Pipe>, DB::QueryPipeline&) @ 0xd6824c2 in /usr/lib/debug/usr/bin/clickhouse
13. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/optional:224: DB::InterpreterSelectQuery::executeWithProcessors() @ 0xd6464fd in /usr/lib/debug/usr/bin/clickhouse
14. /build/obj-x86_64-linux-gnu/../src/Interpreters/InterpreterSelectWithUnionQuery.cpp:256: DB::InterpreterSelectWithUnionQuery::executeWithProcessors() @ 0xd817c75 in /usr/lib/debug/usr/bin/clickhouse
15. /build/obj-x86_64-linux-gnu/../src/Interpreters/executeQuery.cpp:341: DB::executeQueryImpl(char const*, char const*, DB::Context&, bool, DB::QueryProcessingStage::Enum, bool, DB::ReadBuffer*, bool) @ 0xd956d94 in /usr/lib/debug/usr/bin/clickhouse
16. /build/obj-x86_64-linux-gnu/../src/Interpreters/executeQuery.cpp:588: DB::executeQuery(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, DB::Context&, bool, DB::QueryProcessingStage::Enum, bool, bool) @ 0xd95a735 in /usr/lib/debug/usr/bin/clickhouse
17. /build/obj-x86_64-linux-gnu/../programs/server/TCPHandler.cpp:252: DB::TCPHandler::runImpl() @ 0x9256a98 in /usr/lib/debug/usr/bin/clickhouse
18. /build/obj-x86_64-linux-gnu/../programs/server/TCPHandler.cpp:1244: DB::TCPHandler::run() @ 0x9257a70 in /usr/lib/debug/usr/bin/clickhouse
19. /build/obj-x86_64-linux-gnu/../contrib/poco/Net/src/TCPServerConnection.cpp:57: Poco::Net::TCPServerConnection::start() @ 0x1057801b in /usr/lib/debug/usr/bin/clickhouse
20. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/atomic:856: Poco::Net::TCPServerDispatcher::run() @ 0x105784ab in /usr/lib/debug/usr/bin/clickhouse
21. /build/obj-x86_64-linux-gnu/../contrib/poco/Foundation/include/Poco/Mutex_POSIX.h:59: Poco::PooledThread::run() @ 0x106f6fd6 in /usr/lib/debug/usr/bin/clickhouse
22. /build/obj-x86_64-linux-gnu/../contrib/poco/Foundation/include/Poco/AutoPtr.h:223: Poco::ThreadImpl::runnableEntry(void*) @ 0x106f23d0 in /usr/lib/debug/usr/bin/clickhouse
23. start_thread @ 0x76db in /lib/x86_64-linux-gnu/libpthread-2.27.so
24. /build/glibc-OTsEL5/glibc-2.27/misc/../sysdeps/unix/sysv/linux/x86_64/clone.S:97: __clone @ 0x12188f in /usr/lib/debug/lib/x86_64-linux-gnu/libc-2.27.so
, Stack trace (when copying this message, always include the lines below):

0. /build/obj-x86_64-linux-gnu/../contrib/poco/Foundation/src/Exception.cpp:27: Poco::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int) @ 0x1065a1f0 in /usr/lib/debug/usr/bin/clickhouse
1. /build/obj-x86_64-linux-gnu/../src/Common/Exception.cpp:32: DB::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int) @ 0x914dd3d in /usr/lib/debug/usr/bin/clickhouse
2. /build/obj-x86_64-linux-gnu/../src/IO/WriteBufferFromVector.h:70: DB::readException(DB::ReadBuffer&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) @ 0x9186d89 in /usr/lib/debug/usr/bin/clickhouse
3. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:3028: DB::Connection::receiveException() @ 0xdea92df in /usr/lib/debug/usr/bin/clickhouse
4. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:2608: DB::Connection::receivePacket() @ 0xdeaf295 in /usr/lib/debug/usr/bin/clickhouse
5. /build/obj-x86_64-linux-gnu/../src/Client/MultiplexedConnections.cpp:253: DB::MultiplexedConnections::receivePacket() @ 0xdebfe9a in /usr/lib/debug/usr/bin/clickhouse
6. /build/obj-x86_64-linux-gnu/../src/DataStreams/RemoteBlockInputStream.cpp:244: DB::RemoteBlockInputStream::readImpl() @ 0xd2697b4 in /usr/lib/debug/usr/bin/clickhouse
7. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/vector:1003: DB::IBlockInputStream::read() @ 0xd2466dd in /usr/lib/debug/usr/bin/clickhouse
8. /build/obj-x86_64-linux-gnu/../programs/benchmark/Benchmark.cpp:380: DB::Benchmark::execute(std::__1::vector<std::__1::shared_ptr<PoolBase<DB::Connection>::Entry>, std::__1::allocator<std::__1::shared_ptr<PoolBase<DB::Connection>::Entry> > >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, unsigned long) @ 0x92e2713 in /usr/lib/debug/usr/bin/clickhouse
9. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/atomic:840: DB::Benchmark::thread(std::__1::vector<std::__1::shared_ptr<PoolBase<DB::Connection>::Entry>, std::__1::allocator<std::__1::shared_ptr<PoolBase<DB::Connection>::Entry> > >&) @ 0x92e3077 in /usr/lib/debug/usr/bin/clickhouse
10. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/atomic:856: ThreadPoolImpl<ThreadFromGlobalPool>::worker(std::__1::__list_iterator<ThreadFromGlobalPool, void*>) @ 0x91765eb in /usr/lib/debug/usr/bin/clickhouse
11. /build/obj-x86_64-linux-gnu/../src/Common/ThreadPool.h:159: ThreadFromGlobalPool::ThreadFromGlobalPool<void ThreadPoolImpl<ThreadFromGlobalPool>::scheduleImpl<void>(std::__1::function<void ()>, int, std::__1::optional<unsigned long>)::'lambda1'()>(void&&, void ThreadPoolImpl<ThreadFromGlobalPool>::scheduleImpl<void>(std::__1::function<void ()>, int, std::__1::optional<unsigned long>)::'lambda1'()&&...)::'lambda'()::operator()() const @ 0x91773ba in /usr/lib/debug/usr/bin/clickhouse
12. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/atomic:856: ThreadPoolImpl<std::__1::thread>::worker(std::__1::__list_iterator<std::__1::thread, void*>) @ 0x91754ab in /usr/lib/debug/usr/bin/clickhouse
13. /build/obj-x86_64-linux-gnu/../contrib/libcxx/include/memory:2615: void* std::__1::__thread_proxy<std::__1::tuple<std::__1::unique_ptr<std::__1::__thread_struct, std::__1::default_delete<std::__1::__thread_struct> >, void ThreadPoolImpl<std::__1::thread>::scheduleImpl<void>(std::__1::function<void ()>, int, std::__1::optional<unsigned long>)::'lambda1'()> >(void*) @ 0x9173993 in /usr/lib/debug/usr/bin/clickhouse
14. start_thread @ 0x76db in /lib/x86_64-linux-gnu/libpthread-2.27.so
15. /build/glibc-OTsEL5/glibc-2.27/misc/../sysdeps/unix/sysv/linux/x86_64/clone.S:97: __clone @ 0x12188f in /usr/lib/debug/lib/x86_64-linux-gnu/libc-2.27.so
 (version 20.5.1.3327 (official build))
```

Reincarnation of 
https://github.com/ClickHouse/ClickHouse/issues/6546
https://github.com/ClickHouse/ClickHouse/issues/7100

### PR DESCRIPTION
**Pull request by KochetovNicolai**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Bug Fix

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fix error `Cannot convert column because it is constant but values of constants are different in source and result` for remote queries which use deterministic functions in scope of query, but not deterministic between queries, like `now()`, `now64()`, `randConstant()`. Fixes #11327

### CODE REVIEW

File: src/Processors/QueryPlan/ConvertingStep.h [hunk 1]
```diff
@@ -18,6 +18,8 @@ class ConvertingStep : public ITransformingStep
 
 private:
     Block result_header;
+    /// Do not check that constants are same. Use value from result_header.
```
Comment (alexey-milovidov) [inline src/Processors/QueryPlan/ConvertingStep.h:21]: This is a comment about what it does but there is no comment about why it is needed.

Comment (KochetovNicolai) [reply src/Processors/QueryPlan/ConvertingStep.h:21]: Added comment.

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
> 00960_live_view_watch_events_live

- fixed in #13108

**Comment by alexey-milovidov** [comment]
Don't wait for performance test.

**Pull request merged**